tests/infer/test_autoguide.py

```diff
@@ -543,6 +543,13 @@ def model():
     if auto_class is AutoLaplaceApproximation:
         guide = guide.laplace_approximation()
 
+    if hasattr(auto_class, "get_posterior"):
+        posterior = guide.get_posterior()
+        posterior_scale = posterior.variance[-1].sqrt()
+        q = guide.quantiles([0.158655, 0.8413447])
```

Comment:
i can replace with this if you prefer
```
guide.quantiles(dist.Normal(0, 1).cdf(torch.tensor([-1.0, 1.0])).data.numpy().tolist())
```

clunky because `quantiles` wraps in a tensor which raises a user warning if the input is a tensor....

```
UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
```